Cargo.toml

```diff
@@ -26,3 +26,4 @@ maia = { git = "https://github.com/comit-network/maia", rev = "a963084189e1598ca
 maia-core = { git = "https://github.com/comit-network/maia", rev = "e419cb2cf19bed6ec53eaa3672408a6205a9b705", package = "maia-core" } # Unreleased version, pinned before maia's breaking change
 xtra_productivity = { git = "https://github.com/comit-network/xtra-productivity" } # Unreleased
 electrum-client = { git = "https://github.com/da-kami/rust-electrum-client/", branch = "fix/ignore-empty-lines" }
+bdk = { git = "https://github.com/coblox/bdk", branch = "sqlite-version-bumps" }
```

Comment:
Why not a fork on itchysats?

Reply:
No particular reason and we had a fork on coblox already.